### DISCUSSION TITLE
New installation missing module urwid

### DISCUSSION THREAD

**Discussion by jeagh** [mfncooper/paracon#13] [Q&A]
I'm missing something that's probably very basic.

I used the follow command to download pycon:
git clone https://github.com/mfncooper/paracon

This resulted in parcon.py  being found in the  /pycon/pycon directory. There isn't a  pycon.pyz file.

python3 paracon.py returns a missing module urwid error. urwid.py is in the same directory.
Ubuntu has python3-urwid installed by default.

If I need to use pip to install urwid then I'll need to set up a virtual environment. If that's the case I'm not sure that dire wolf will be detected because it's installed system wide.

**Comment by mfncooper** [comment]
If you just want to _run_ Paracon, then you should simply download the .pyz file from the [Releases](https://github.com/mfncooper/paracon/releases) area, as described in the [installation instructions](https://github.com/mfncooper/paracon?tab=readme-ov-file#installation) in the README. Since Paracon is built as a .pyz file, which is self-contained, you do not need to create a virtual environment, and you do not need to download or install any dependencies.

If you're trying to build Paracon from source code, then you'll need to run the [build_zipapp.sh](https://github.com/mfncooper/paracon/blob/main/build_zipapp.sh) shell script to do that. The script will download the dependencies, including urwid. But you shouldn't need the source code unless you're trying to debug something or modify the code.

**Reply by jeagh** [reply]
Thank you, I knew the answer would be obvious to anyone else.
